src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -264,7 +264,7 @@ public enum FlowMode {
                 MIN_BASE_QUALITY_SCORE_SHORT_NAME, "0",
                 FILTER_ALLELES, "true",
                 FILTER_ALLELES_SOR_THRESHOLD, "3",
-                FLOW_ASSEMBLY_COLLAPSE_HMER_SIZE_LONG_NAME, "12",
+                FLOW_ASSEMBLY_COLLAPSE_HMER_SIZE_LONG_NAME, String.valueOf(AssemblyBasedCallerUtils.DETERMINE_COLLAPSE_THRESHOLD),
```

Comment:
Does this change the default for STANDARD/ADVANCED flow mode for older samples that don't have the mc tag?